cmd/image-builder/logging.go

```diff
@@ -0,0 +1,66 @@
+package main
+
+import (
+	"context"
+	"strings"
+
+	"github.com/labstack/echo/v4"
+	"github.com/labstack/gommon/random"
+	"github.com/osbuild/image-builder/internal/common"
+	"github.com/sirupsen/logrus"
+)
+
+type ctxKey int
+
+const (
+	requestIdCtx ctxKey = iota
+)
+
+// Use request id from the standard context and add it to the message as a field.
+type ctxHook struct {
+}
+
+func (h *ctxHook) Levels() []logrus.Level {
+	return []logrus.Level{
+		logrus.DebugLevel,
+		logrus.InfoLevel,
+		logrus.WarnLevel,
+		logrus.ErrorLevel,
+		logrus.FatalLevel,
+		logrus.PanicLevel,
+	}
+}
+
+func (h *ctxHook) Fire(e *logrus.Entry) error {
```

Comment:
@lzap @schuellerf could this be a culprit of https://github.com/osbuild/image-builder/issues/1248 ?

Reply:
Likely: https://github.com/osbuild/image-builder/pull/1251